docs/DeveloperGuide.md

```diff
@@ -943,6 +982,62 @@ The above sequence diagram omits details such as internal method calls to `GetBu
 The only difference between the negative and non-negative messages is that the negative message places the `-` symbol before the `$` so `-$xx.xx` whereas the non-negative message does not contain the `-` symbol
 </div>
 
+#### Summary Statistics
+
+**Overview**
+
+The `summary` command creates a pop-up summary window that contains summary plots to provide some basic analytics about
+the user's expense data. 
+
+The activity diagram of the `summary` command is given below:
+
+<img src="images/unicash/SummaryActivityDiagram.png" width="700" />
+
+Let's first go over the sequence of events that occur within the Ui component first to understand the code that performs
+the activities shown in the activity diagram above. The sequence diagram below documents this:
+
+<img src="images/unicash/SummaryUiSequenceDiagram.png" width="700" />
```

Comment:
I'm not too sure if we need so much detail about every method call in between, since the green background makes things a little hard to read so having so much detail there might be a readability issue. Could also consider simplify the call arrow to be "Prepare data" and then explain what that means in the `Details` section

Reply:
Fixed! I named the call arrow "get expense data", but thanks for the suggestion